rules/use-one-version-manager-supporting-multiple-softwares/rule.md

```diff
@@ -0,0 +1,60 @@
+---
+type: rule
+archivedreason: 
+title: Do you know to use one version manager supporting multiple softwares?
+guid: 6b7b2c3a-e363-4402-b1ee-460fd16125fb
+uri: use-one-version-manager-supporting-multiple-softwares
+created: 2023-07-21T21:54:27.0000000Z
+authors:
+- title: John Xu
+  url: https://ssw.com.au/people/john-xu
+related: []
+redirects:
+- do-you-know-to-use-one-version-manager-supporting-multiple-softwares
+
+---
+
+Use one version manager supporting multiple softwares.
+
+<!--endintro-->
+
+![](ring.png)
+
+Every programming language or framework build upon it has its own version managers in community, such as NVM for Node.js, RVM for Ruby, and Pyenv for Python.
+
+However, it is not a good idea to use different version managers for different softwares. It is better to use one version manager supporting multiple softwares.
+
+
+### Option A: asdf (recommended)
+
+![](asdf.png)
+
+[asdf - manage all your runtime versions with one tool](https://asdf-vm.com/#/)
```

Comment:
1. Add link only on the software name "asdf"
2. Make the 1st letter Uppercase after the dash

Reply:
Done